processor/src/main/resources/org/mapstruct/ap/internal/model/source/builtin/DateToSqlDate.ftl

```diff
@@ -0,0 +1,27 @@
+<#--
+
+     Copyright 2012-2016 Gunnar Morling (http://www.gunnarmorling.de/)
+     and/or other contributors as indicated by the @authors tag. See the
+     copyright.txt file in the distribution for a full listing of all
+     contributors.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+private java.sql.Date ${name}( Date date ) {
```

Comment:
Hey, what's the reasoning behind implementing these conversions through methods rather than "built-in conversions" (see `SimpleConversion` and e.g. `JavaLocalDateToStringConversion`). That should also take care of handling the imports (so no FQN needed in generated code unless there is a collision of types with the same simple name).